tests/test_tree.py

```diff
@@ -0,0 +1,66 @@
+from typing import Type
+
+import pytest
+
+from merkle_zeppelin import BinaryTree
+
+
+@pytest.mark.parametrize("elements_number", [1, 2, 3, 4, 5, 6, 9, 1025, 2049])
+def test_good_leafs_number(
+    elements_number: int, max_tree_class: Type[BinaryTree]
+) -> None:
+    # given
+    leafs = [x for x in range(elements_number)]
+
+    # when
+    tree = max_tree_class(leafs)
+
+    # then
+    assert tree._leafs_number == elements_number
+    # assert len(tree.leafs) == leafs_number
```

Comment:
Do you want to leave this assert commented out here?

Reply:
thanks for finding it!
gonna uncomment it

Reply:
or actually it's useless...